oml/models/resnet.py

```diff
@@ -167,7 +173,7 @@ def draw_gradcam(self, image: Union[np.ndarray, TPILImage]) -> Union[np.ndarray,
         if self.remove_fc:
             raise ValueError("This method does not work if there is no FC layer in the model.")
 
-        need_to_convert = type(image) == TPILImage
+        need_to_convert = type(image) != np.ndarray
```

Comment:
I'd suggest a more pythonic way

```python
need_to_convert = not isinstance(image, np.ndarray)
```

And the same for `vit`.

Reply:
okay